vnext/Microsoft.ReactNative.Cxx.UnitTests/JsonJSValueReader.cpp

```diff
@@ -0,0 +1,160 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include "JsonJSValueReader.h"
+
+namespace winrt::Microsoft::ReactNative::Bridge {
+
+//===========================================================================
+// JsonJSValueReader implementation
```

Comment:
>JsonJSValueReader [](start = 3, length = 17)

Would this ever be useful outside of UT code? Wondering if we should share it outside the UT directory.

Reply:
Yes, someone already asked for it too. I will move it out to the Microsoft.ReactNative.Cxx in a future PR.

---
In reply to: [356922710](https://github.com/microsoft/react-native-windows/pull/3760#discussion_r356922710) [](ancestors = 356922710)